src/main/java/dev/latvian/mods/kubejs/core/TagLoaderKJS.java

```diff
@@ -56,6 +58,12 @@ public interface TagLoaderKJS<T> {
 				}
 			}
 
+			for (var e : kjs$getResources().kjs$getServerScriptManager().serverRegistryTags.entrySet()) {
+				for (var tag : e.getValue()) {
+					event.add(tag, new TagEventFilter.ID(e.getKey()));
+				}
```

Comment:
nitpick: the SSM is probably used often enough here that it should be stored as a variable, just for readability's sake